src/CodeGen.fs

```diff
@@ -1072,40 +1076,130 @@ let sampleFSharpClientMember query queryName hasVariables = stringBuffer {
+        sprintf """namespace %s
 
 open Fable.SimpleHttp
 open Fable.SimpleJson
+open Newtonsoft.Json
+open Newtonsoft.Json.Linq
 
 type GraphqlInput<'T> = { query: string; variables: Option<'T> }
 type GraphqlSuccessResponse<'T> = { data: 'T }
 type GraphqlErrorResponse = { errors: %s list }
 
-type %s(url: string, headers: Header list) =
-    new(url: string) = %s(url, [ ])
+type %s(url: string, headers: Header list, settings: JsonSerializerSettings) =
+    new(url: string, settings: JsonSerializerSettings) = %s(url, [ ], settings)
+    new(url: string) = %s(url, [ ], new JsonSerializerSettings ())
 
-%s""" projectName errorType clientName clientName members
-}
+%s""" projectName errorType clientName clientName clientName members
+    }
 
-let sampleFSharpGraphqlClient projectName clientName errorType members = stringBuffer {
-    sprintf """namespace %s
+let private sampleFSharpSystemGraphqlClient projectName clientName errorType members =
+    stringBuffer {
+        sprintf """namespace %s
+
+open System.Net.Http
+open System.Text
+open System.Text.Json
+
+type GraphqlInput<'T> = { query: string; variables: Option<'T> }
+type GraphqlSuccessResponse<'T> = { data: 'T }
+type GraphqlErrorResponse = { errors: %s list }
+
+type %s(url: string, httpClient: HttpClient, options: JsonSerializerOptions) =
+    new(url: string, options: JsonSerializerOptions) = %s(url, new HttpClient(), options)
+    new(url: string) = %s(url, new HttpClient(), new JsonSerializerOptions())
+
+    %s""" projectName errorType clientName clientName clientName members
+    }
+
+let private sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members =
+    stringBuffer {
+        sprintf """namespace %s
 
-open Fable.Remoting.Json
 open Newtonsoft.Json
 open Newtonsoft.Json.Linq
+open Fable.Remoting.Json
 open System.Net.Http
 open System.Text
 
 type GraphqlInput<'T> = { query: string; variables: Option<'T> }
 type GraphqlSuccessResponse<'T> = { data: 'T }
 type GraphqlErrorResponse = { errors: %s list }
 
-type %s(url: string, httpClient: HttpClient) =
-    let converter = FableJsonConverter() :> JsonConverter
-    let settings = JsonSerializerSettings(DateParseHandling=DateParseHandling.None, Converters = [| converter |])
+type %s(url: string, httpClient: HttpClient, settings: JsonSerializerSettings) =
+    let fableJsonConverter = FableJsonConverter() :> JsonConverter
+    let settings =
+        settings.Converters.Insert (0, fableJsonConverter)
+        settings
+    let settings = JsonSerializerSettings(DateParseHandling=DateParseHandling.None, Converters = settings.Converters)
 
-    new(url: string) = %s(url, new HttpClient())
+    new(url: string, settings: JsonSerializerSettings) = %s(url, new HttpClient(), settings)
 
-%s""" projectName errorType clientName clientName members
-}
+    new(url: string) = %s(url, new HttpClient(), new JsonSerializerSettings())
+
+    %s""" projectName errorType clientName clientName clientName members
+    }
```

Comment:
Try using string interpolation. I suppose we don't need stringBuffer here.
It is better to cache values via let prior to string interpolation.